docs/deployment_deep_research.md

```
@@ -0,0 +1,147 @@
+
+1. **Create Resource Group**:  
+   ```bash
+   az group create --name roomies-rg --location centralindia
+   ```
+2. **Provision PostgreSQL**:  
+   ```bash
+   az postgres flexible-server create \
+     --resource-group roomies-rg --name roomies-db --location centralindia \
+     --admin-user adminuser --admin-password '<password>' \
+     --sku-name GP_Burstable_B1ms --storage-size 32
+   ```  
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)